css/main.css

```diff
@@ -44,7 +44,17 @@ a { text-decoration:none;}
     opacity: 0;
     filter: alpha(opacity=0);
   }
-  
+  .page-title ul li i:first-child{
+    margin-left:20px;
+  }
+  .page-title ul li{
+    display:inline-block;
+    padding-right:15px;
+  }
+.page-title ul li i{
```

Comment:
Your blocks have to be aligned, indentation is VERY important to read the code easily. In line 50 you have 2 spaces, and in this line 54 you don't have any space, it's supposed to be uniform. 👁